packages/components/src/internal/app/constants.ts

```diff
@@ -138,6 +138,8 @@ export enum ProductFeature {
     Workflow = 'Workflow',
 }
 
+const LKB_SAMPLES_EXCLUDED_EXPORT_COLUMNS = ['flag', 'Ancestors', 'RawAmount', 'RawUnits']; // StorageStatus as well?
```

Comment:
Yes, StorageStatus as well. Users can't provide this value so it isn't useful in the template.

I know this naming is following what was already there, but since these constants are used for specifying template exclusions, perhaps the names should reflect that: `LKB_SAMPLES_TEMPLATE_EXCLUDED_COLUMNS`?